package.json

```diff
@@ -73,7 +73,7 @@
 		"@babel/traverse": "7.4.5",
 		"@octokit/rest": "16.26.0",
 		"@storybook/addon-a11y": "5.2.1",
-		"@storybook/addon-docs": "5.3.0-alpha.2",
```

Comment:
We want to be on the latest for the new features.

Reply:
I couldn't find any explanation of why we would want to use a non-stable version of this addon. Everything else is on the latest published version which is what you would expect when using external tooling. I'd like to learn why do you think it's valuable to use `alpha` release instead.

Reply:
It had a lot less bugs and more features, but I see these were stabilized in `5.2.2`. Can we use that or `5.2.3` (`latest`)?

Reply:
Sure, thing. `5.2.3` for all those dependencies sounds great 👍